ramls/department.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Department",
+  "description": "Department object schema",
+  "javaType": "org.folio.rest.jaxrs.model.Department",
+  "additionalProperties": false,
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "A UUID identifying this department",
+      "$ref": "raml-util/schemas/uuid.schema",
+      "example": "f973c3b6-85fc-4d35-bda8-f31b568957bf"
+    },
+    "type": {
```

Comment:
mod-users doesn't follow JSON API standards for content structure.
it's not required to have 
* id
* type
* attributes